pages/pipelines/governance.md

```diff
@@ -0,0 +1,3 @@
+# Governance overview
+
+Buildkite cares deeply about making sure enterprise customers have their compliance needs met. These features are aimed to help with the auditing processes in your company be it specific for your industry or wide spread compliance standards like SOC.
```

Comment:
This helps the page look more worthwhile, but it doesn't say all that much:

```suggestion
Governance plays a crucial role in CI/CD tools, particularly for anyone operating in regulated industries or handling sensitive data. It encompasses policies, processes, and controls to ensure practices align with the following:

- Industry-specific regulations.
- Internal policies.
- Widely recognized compliance standards.

These standards assess the security, availability, processing integrity, confidentiality, and privacy of information systems.

Buildkite understands the importance of meeting compliance and auditing requirements. [Pipeline templates](/docs/pipelines/templates) help you control the step configurations used across pipelines in your organization.

```

What do you think?

Reply:
Note, I didn't include SOC specifically because I couldn't see an obvious link to include for it. I'm happy to put it in if you think it's important, and even happier if we can put it in with a link to an explanation.

Reply:
Thanks @mbelton-buildkite, myself and @ozdenyilmaz think this is a great improvement 👍